ddtrace/span.py

```diff
@@ -228,6 +232,12 @@ def set_traceback(self, limit=20):
             tb = ''.join(traceback.format_stack(limit=limit + 1)[:-1])
             self.set_tag(errors.ERROR_STACK, tb)  # FIXME[gabin] Want to replace "error.stack" tag with "python.stack"
 
+    def remove_traceback(self):
```

Comment:
this is a duplication of the method below. Both includes `self.remove_tag(errors.ERROR_STACK)` so it will be executed twice (the second will do nothing). I think you can remove it. I would say the API should contain only internal methods to:
* safely remove a tag (`_remove_tag`)
* safely remove the entire exc_info (`_remove_exc_info `)